src/filesystem/impls/appshell/node/node_modules/fsevents_win/fsevents_win.cpp

```diff
@@ -337,7 +339,7 @@ DWORD WINAPI NodeFSEvents::Run(LPVOID lpData)
 	{
 		// iterate watching for directory changes until we're signalled to stop
 		BOOL bContinue = TRUE;
```

Comment:
@mfatekho I think there's still a logic error here.  If the `ReadDirectoryChangesW()` call below fails, then `bContinue` is assigned the value of `FALSE`.  As a result, on the next iteration of the loop, the `while` condition below in line 342 will *still* be false, and we'll still get kicked out of the loop.

I'd partially agree with @busykai 's [comment](https://github.com/adobe/brackets/pull/12353/files#r59280730) above, and propose removing `bContinue` from the `while` loop condition expression in line 342.  (Although, per the above discussion, don't entirely remove the `bContinue` variable b/c it'll still be needed below to capture the result of `ReadDirectoryChangesW()`.